dashboard/src/App.js

```diff
@@ -43,24 +47,33 @@ class App extends Component {
   reloadOperators = async() => {
     try {
       const operators = await api.get('/api/operators');
-      this.setState({operators, error: undefined});
+      this.setState({
+        operators,
+        error: undefined
+      });
     } catch (e) {
-      this.setState({error: e.message});
+      this.setState({
+        error: e.message
+      });
+      if (IsUnauthorized(e)) {
+        this.props.doLogout();
+      }
     }
     this.props.setTimeout(this.reloadOperators, 10000);
   }
 
   render() {
     if (this.state.operators) {
-      return <OperatorsView 
+      return <OperatorsView
         error={this.state.error}
-        deployment={this.state.operators.deployment} 
-        pod={this.state.operators.pod} 
-        namespace={this.state.operators.namespace} 
-      />;
+        deployment={this.state.operators.deployment}
+        storage={this.state.operators.storage}
+        pod={this.state.operators.pod}
+        namespace={this.state.operators.namespace}
+        />;
     }
     return (<LoadingView/>);
   }
 }
 
-export default ReactTimeout(App);
+export default ReactTimeout(withAuth(App));
```

Comment:
FYI this is why HOCs have been losing some traction to render props recently: you're now injecting props from two places. This is probably fine for the time being but it can get a bit confusing as the complexity grows.